packages/synthetics-sdk-api/proto/synthetic_response.proto

```diff
@@ -175,13 +175,11 @@ message BrokenLinksResultV1 {
     // Maximum number of times to retry a link that does not return the
     // “expected_status_code”.
     optional int64 max_retries = 7;
-    // Maximum number of times to follow redirects for a link, defaults to following all redirects.
-    optional int64 max_redirects = 8;
     // HTML element to wait for before scraping links on origin_uri.
     // Method documentation:
     // https://pptr.dev/api/puppeteer.page.waitforselector. Type documentation:
     // https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors
-    string wait_for_selector = 9;
+    string wait_for_selector = 8;
```

Comment:
you shouldnt re-index proto fields, instead mark max_redirects as deprecated, or perhaps dont do anything with it at all right now.

Reply:
Since it is has not been publicly released I thought it would be fine to just redefine the proto. 

I definitely think we should remove it from the proto but we could leave wait_for_selector as 9 in preparation for potentially adding max_redirects back in the future?